### PR TITLE
Mirror Effect with hover effects

### PR DESCRIPTION
**Pull request by naps62**

### CODE REVIEW

File: Components/Mirror.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+
+import '../css/Components/Mirror';
+
+export default class Mirror extends Component {
+  componentDidMount = () => {
+    document.querySelectorAll('.Mirror-original [data-mirror-hover]').forEach(element => {
```
Comment (joaojusto) [inline Components/Mirror.js]: `...).forEach(this.mirrorEventsOn)`

File: pages/index.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ export default class Index extends React.Component {
           ]}
         />
 
-        <Hero/>
+          <Hero/>
```
Comment (joaojusto) [inline pages/index.js]: Why? 🔔

**Review by joaojusto — APPROVED** [review]
Take a look at my comments and :shipit:

File: Components/Mirror.js (hunk 1 above)
Comment (naps62) [reply Components/Mirror.js]: fixed

**Pull request merged**